projects/storefrontlib/src/cms-components/asm/services/asm-enabler.service.ts

```diff
@@ -77,13 +71,6 @@ export class AsmEnablerService {
    * Adds the ASM UI by using the `cx-storefront` outlet.
    */
   private addUi(): void {
```

Comment:
I know it's not on you, but I'd prefer to make all private method protected here.

Reply:
Yes approved